### PR TITLE
Fix/3542 margins between paragraphs in comments

### PR DESCRIPTION
**Pull request by raphj**
#3542 

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [X] If there are modifications in the Tracim configuration files (eg. `development.ini`), make sure it is documented in `backend/doc/setting.md` and notify relevant people of this change

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/Timeline/Comment.styl [hunk 1]
```diff
@@ -23,8 +23,6 @@
     &__text
       margin-left 20px
       margin-top 10px
-  p
-    margin 0
-    & img
+  p img
       max-width 100%
```
Comment (gbonaspetti) [inline frontend_lib/src/component/Timeline/Comment.styl:27]: the tab is right here? it looks a little big

File: functionnal_tests/cypress/support/db_commands.js [hunk 2]
```diff
@@ -187,6 +187,14 @@ Cypress.Commands.add('createThread', (title, workspaceId, parentId = null) => {
     .then(handleUndefinedResponse)
 })
 
+Cypress.Commands.add('createComment', (workspaceId, contentId, rawContent) => {
+  let url = `/api/workspaces/${workspaceId}/contents/${contentId}/comments`
+  let data = { raw_content: rawContent }
+  cy
+    .request('POST', url, data)
+    .then(handleUndefinedResponse)
```
Comment (gbonaspetti) [inline functionnal_tests/cypress/support/db_commands.js:195]:
```suggestion
  cy.request('POST', url, data)
    .then(handleUndefinedResponse)
```

File: frontend_lib/src/component/Timeline/Comment.styl (hunk 1 above)
Comment (raphj) [reply frontend_lib/src/component/Timeline/Comment.styl:27]: Indeed, I forgot to deindent this line

Comment (raphj) [reply frontend_lib/src/component/Timeline/Comment.styl:27]:
```suggestion
  p img
    max-width 100%
```

File: frontend_lib/src/component/Timeline/Comment.styl [hunk 3]
```diff
@@ -23,8 +23,6 @@
     &__text
       margin-left 20px
       margin-top 10px
-  p
-    margin 0
-    & img
-      max-width 100%
+  p img
+    max-width 100%
       height auto
```
Comment (raphj) [inline frontend_lib/src/component/Timeline/Comment.styl:28]:
```suggestion
    height auto
```

**Review by gbonaspetti — APPROVED** [review]

File: frontend_lib/src/component/Timeline/Comment.styl [hunk 4]
```diff
@@ -23,8 +23,6 @@
...
       margin-left 20px
       margin-top 10px
-  p
-    margin 0
-    & img
-      max-width 100%
-      height auto
+  p img
+    max-width 100%
+    height auto
```
Comment (raphj) [inline frontend_lib/src/component/Timeline/Comment.styl:28]:
```suggestion
    p img
      max-width 100%
      height auto
```

**Comment by gbonaspetti** [comment]
Tested on branch.

**Pull request merged**